lib/web_ui/lib/src/engine/profiler.dart

```diff
@@ -71,7 +71,7 @@ class Profiler {
     return Profiler._instance ??= Profiler._();
   }
 
-  static Profiler get instance {
+  static Profiler? get instance {
```

Comment:
This never returns null. If `_instance` is null, it throws.

Reply:
Done.